pages/help/help_domain-requests.md

```diff
@@ -2,18 +2,24 @@
 title: Domain requests
 permalink: /help/domain-requests/
 layout: layouts/info-page
-excerpt: Learn how to request a .gov domain
+excerpt: Get help with domain requests
 tags: help
 eleventyNavigation:
   key: help
   order: 2 
 ---
 
-If you’re ready to request your .gov domain then let's get started. You don’t have to complete the process in one session. You can save what you enter and come back to it when you’re ready. 
+Get help with domain requests. 
 
-<button class="usa-button">Start your .gov domain request</button>
+- [What to do before you request your .gov domain](#before-you-request-your-.gov-domain)
+- [Change a domain request you already submitted](#change-a-domain-request-you-already-submitted)
+- [Check the status of your domain request](#check-the-status-of-your-domain-request)
+- [Withdraw your domain request](#withdraw-your-domain-request)
+
+## Request your .gov domain
+Ready to request your .gov domain? 
 
-If you'd like more information, read on for help with your .gov domain request.
+<button class="usa-button">Start your .gov domain request</button>
```

Comment:
Similar pattern to our other pages, let's make this a a link with a class added `{.usa-button}`. I think we can now also link it to the form?

Reply:
Thanks! I'll wait on adding that link. Once we decide what it should be we can update it everywhere.